README.md

```diff
@@ -520,6 +520,63 @@ This last conditional block is simply to ignore the unneeded license files, base
 
 > :warning: Note that `include` and `exclude` are still mutually exclusive even if they are in different, but included, conditional sections.
 
+## Useful for template authors
+
+> Available since version [0.9.0]
+
+As a template author you're probably concerned about successful builds of your template?
+
+Imagine a couple of month after your first template release, some new versions of any dependencies would break your template, and you would not even be aware of it?
+
+The answer to this question is a vital build pipeline for your template project. This challenge got now much simpler to solve with the new official [GitHub Action cargo-generate][gh/action].
```

Comment:
Didn't know of the md link style with 2 following `[xxx][yyy]`... only the normal `[xxx](link)` or `[xxx]` and then later `[xxx]: link`

Guess I now know how to rename the link in the latter style... cool 👍🏽